src/features/DisplayControls/NursingTasks/components/UpdateNursingTasks.jsx

```diff
@@ -114,101 +221,193 @@ const UpdateNursingTasks = (props) => {
+                      invalidText={invalidTimeText24Hour}
                       light={true}
                     />
-                    {showErrors && errors[medicationTask.uuid] && (
-                      <div className={"error"}>
-                        <FormattedMessage
-                          id={"NOTES_ERROR_MESSAGE"}
-                          defaultMessage={"Please enter notes"}
-                        />
-                      </div>
-                    )}
+                    <div className={"notes-text-area"}>
+                      <TextArea
+                        labelText={
+                          <Title
+                            text={"Notes"}
+                            isRequired={
+                              tasks[medicationTask.uuid].isTimeOutOfWindow
+                            }
+                          />
+                        }
+                        onChange={(e) => {
+                          handleNotes(e, medicationTask.uuid);
+                        }}
+                        maxCount={250}
+                        rows={1}
+                        cols={50}
+                        light={true}
+                      />
+                      {showErrors && errors[medicationTask.uuid] && (
+                        <div className={"error"}>
+                          <FormattedMessage
+                            id={"NOTES_ERROR_MESSAGE"}
+                            defaultMessage={"Please enter notes"}
+                          />
+                        </div>
+                      )}
+                    </div>
                   </div>
-                </div>
-              )}
-            </div>
-          );
-        })}
-      </div>
-      <SaveAndCloseButtons
-        onSave={() => {
-          updateShowErrors(true);
-        }}
-        onClose={() => {
-          updateNursingTasksSlider(false);
-        }}
-        isSaveDisabled={isSaveDisabled}
-      />
-    </SideBarPanel>
+                )}
+              </div>
+            );
+          })}
+        </div>
+        <Modal
+          open={isAnyMedicationAdministered && openConfirmationModal}
+          onRequestClose={closeModal}
+          onSecondarySubmit={closeModal}
+          preventCloseOnClickOutside={true}
+          modalHeading={
+            <FormattedMessage
+              id={"NURSING_TASKS_CONFIRMATION"}
+              defaultMessage={"Please confirm your nursing tasks"}
+            />
+          }
+          primaryButtonText={
+            <FormattedMessage
+              id={"DRUG_CHART_MODAL_SAVE"}
+              defaultMessage={"Save"}
+            />
+          }
+          secondaryButtonText={
+            <FormattedMessage
+              id={"DRUG_CHART_MODAL_CANCEL"}
+              defaultMessage={"Cancel"}
+            />
+          }
+          onRequestSubmit={handlePrimaryButtonClick}
+        >
+          <div className="divider"></div>
```

Comment:
As this div tag is only adding a horizontal line separator, try use the <hr> tag for separation